src/panel/category/CategoryPanel.jsx

```diff
@@ -5,13 +5,13 @@ import Panel from 'src/components/ui/Panel';
 import PoiItemList from './PoiItemList';
 import PoiItemListPlaceholder from './PoiItemListPlaceholder';
 import CategoryPanelError from './CategoryPanelError';
-import CategoryPanelHeader from './CategoryPanelHeader';
 import Telemetry from 'src/libs/telemetry';
 import nconf from '@qwant/nconf-getter';
 import IdunnPoi from 'src/adapters/poi/idunn_poi';
 import { getVisibleBbox } from 'src/panel/layouts';
 import { fire, listen, unListen } from 'src/libs/customEvents';
 import { boundsFromFlatArray, parseBboxString, boundsToString } from 'src/libs/bounds';
+import { sources } from '../../../config/constants.yml';
```

Comment:
To avoid these long chains of `../` and always having to know where you are in the tree, you can import from `'config/constants.yml'`, as there is a webpack alias on `config` like we use for `src`.

Reply:
sorry, my IDE added this line automatically and I didn't catch it